docs/api/tray.md

```diff
@@ -193,6 +193,19 @@ Sets the title displayed aside of the tray icon in the status bar.
 
 Sets when the tray's icon background becomes highlighted (in blue).
 
+**Note:** You can use `highlightMode` with a [`BrowserWindow`](browser-window.md)
+by toggling between `'never'` and `'always'` modes when the window visibility
+changes.
+
+```js
+win.on('show', () => {
```

Comment:
This is failing the build because these variables are never defined and #6603 runs `standard` on all snippets so it is complaining.

Adding the following at the beginning of the snippet should address this:

```js
let win = new BrowserWindow({width: 800, height: 600})
let tray = new Tray('/path/to/my/icon')
```